src/main/java/com/icthh/xm/ms/dashboard/repository/impl/ConfigDashboardRepository.java

```diff
@@ -134,4 +139,14 @@ private <S extends Dashboard> void validate(S dashboard) {
             throw new ValidationException(constraintViolations.toString());
         }
     }
+
+    @Override
+    public Page<Map<String, Object>> findAllAudits(Pageable pageable) {
+        throw new NotYetImplementedException();
```

Comment:
Why do you use so specific exception from Hibernate? Let's use NotImplementedException from apache commons as we use usually